chia/wallet/wallet_node.py

```diff
@@ -461,10 +462,10 @@ async def _process_new_subscriptions(self):
                     # This can take a VERY long time, because it might trigger a long sync. It is OK if we miss some
                     # subscriptions or state updates, since all subscriptions and state updates will be handled by
                     # long_sync (up to the target height).
-                    request: wallet_protocol.NewPeakWallet = item.data[0]
```

Comment:
mypy: `request` was "redefined" here.

Reply:
Maybe we should rename both for consistency?